controllers/teamdetail.controller.js

```diff
@@ -36,6 +36,47 @@ function getTeamDetail(req, res, next) {
     }
 }
 
+function getGithubTeamDetail(req, res, next) {
+    let team = req.query.team;
+    let startDate = req.query.startDate ? req.query.startDate : utils.parseTimestampToDate(defaultStartDate);
+    let endDate = req.query.endDate ? req.query.endDate : utils.parseTimestampToDate(defaultEndDate);
+
+    if (typeof team !== 'undefined' && team !== null){
+      let githubBackendUrl = utils.getGithubBackendUrl();
+      return axios
+          .get(`${githubBackendUrl}/team-impact`, {
+              params: {
+                  team: team,
+                  startDate: startDate,
+                  endDate: endDate
+              }
+          })
+          .then(response => {
+              console.log(response)
+              res.json({
+                impact: {
+                  teamImpact: response.data.teamImpact,
+                  avgTeamImpact: response.data.avgTeamImpact
+                }
+              })
+          })
+          .catch(error => {
+            console.log(error)
+              res.json({
+                impact: {
+                  teamImpact: 0,
+                  avgTeamImpact: 0
+                }
+              })
+          })
+    } else {
+        res.json({
+            error : "Team is not provided"
+        });
+    }
```

Comment:
This check can be simplified to:
```suggestion
if (!team) {
        next(new Error("URL parameter 'team' was not provided in the request"));
    }

    let githubBackendUrl = utils.getGithubBackendUrl();
    return axios
        .get(`${githubBackendUrl}/team-impact`, {
            params: {
                team: team,
                startDate: startDate,
                endDate: endDate
            }
        })
        .then(response => {
            console.log(response)
            res.json({
                impact: {
                    teamImpact: response.data.teamImpact,
                    avgTeamImpact: response.data.avgTeamImpact
                }
            })
        })
        .catch(error => {
            console.log(error)
            res.json({
                impact: {
                    teamImpact: 0,
                    avgTeamImpact: 0
                }
            })
        });
```